src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -281,55 +355,92 @@ const AddAppealForm = (): ReactElement => {
                     variant="h6"
                     display="inline"
                     className={classes.selectAll}
+                    onClick={selectAllStatuses}
                   >
                     {t('select all')}
                   </Typography>
-                  <TextField
-                    variant="outlined"
-                    placeholder={t('Select Some Options')}
-                    className={classes.input}
-                    inputProps={{
-                      style: {
-                        padding: 10,
-                      },
-                    }}
-                  />
-                </Box>
-                <Box mt={1} mb={1}>
-                  <Typography variant="h6" display="inline">
-                    {t('Add contacts with the following tag(s):')}
-                  </Typography>
-                  <Typography
-                    variant="h6"
-                    display="inline"
-                    className={classes.selectAll}
-                  >
-                    {t('select all')}
-                  </Typography>
-                  <TextField
-                    variant="outlined"
-                    placeholder={t('Select Some Options')}
-                    className={classes.input}
-                    inputProps={{
-                      style: {
-                        padding: 10,
-                      },
-                    }}
+                  <Autocomplete
+                    multiple
+                    id="tags-standard"
+                    options={contactStatusesFormatted.filter(
+                      ({ value: id1 }) =>
+                        !filterTags.statuses.some(
+                          ({ value: id2 }) => id2 === id1,
+                        ),
+                    )}
+                    getOptionLabel={(option) => option.title}
+                    value={filterTags.statuses}
+                    onChange={({}, values) => handleChange(values, 'statuses')}
```

Comment:
You just throwing the event away? I think the underscore prefix is a better syntax. ESLint shouldn't complain about unused function params that start with an underscore.
```suggestion
                    onChange={(_event, values) => handleChange(values, 'statuses')}
```